ompi/mpi/tool/cvar_read.c

```diff
@@ -37,8 +37,10 @@ int MPI_T_cvar_read (MPI_T_cvar_handle handle, void *buf)
         return MPI_T_ERR_NOT_INITIALIZED;
     }
 
-    if (MPI_PARAM_CHECK && NULL == buf) {
-        return MPI_T_ERR_INVALID;
+    if (MPI_PARAM_CHECK) {
+        if ((NULL == buf) || (NULL == handle)) {
```

Comment:
Should this check for `MPI_T_BIND_NO_OBJECT` here too?

Reply:
no I got confused by the standard.  the MPI_T_BIND_NO_OBJECT is returned from MPI_T_(C/P)VAR_GET_INFO in the ```bind``` argument.  I fixed comments in the changes that aren't correct.

Reply:
MPI_T_BIND_NO_OBJECT should not be used as a value for the ```obj_handle``` arg to the handle alloc methods.